projects/plugins/jetpack/extensions/plugins/publicize/hooks/use-social-media-connections/Readme.md

```diff
@@ -0,0 +1,49 @@
+# useSocialMediaConnections hook
+
+React hook to deal with the social media connections actions.
+You'd like to use together with the [useSelectSocialMedia](../use-select-social-media) selector hook.
```

Comment:
```suggestion
React hook to deal with the social media connections actions.
This hook should be used together with the [useSelectSocialMedia](../use-select-social-media) selector hook.
```
These 2 sentences are a little confusing to me. Do you mean "This hook should be used together with"?